course-preparation/Ansible/group_vars/addis.yml

```diff
@@ -13,6 +13,17 @@ prerequisites:
     - gcc
     - nano
     - joe
+  Ubuntu:
```

Comment:
This probably won't be used. The fact we're using is `ansible_os_family`, which resolves to `Debian` on Ubuntu machines : 

```
becker@serbaggio:~/Ops/AAROC/AnsibleBootCamp/course-preparation/Ansible$ ansible localhost -m setup -a filter="ansible_os_family"
 [WARNING]: Host file not found: /etc/ansible/hosts

 [WARNING]: provided hosts list is empty, only localhost is available

localhost | SUCCESS => {
    "ansible_facts": {
        "ansible_os_family": "Debian"
    }, 
    "changed": false
}
```